src/common/docsEnvironmentController.ts

```diff
@@ -1,22 +1,31 @@
 import vscode from 'vscode';
-import { Environment, EXTENSION_NAME } from '../shared';
+import { Environment, DocsRepoType, EXTENSION_NAME } from '../shared';
 import { EnvironmentChanged } from './loggingEvents';
 import { EventStream } from './eventStream';
 import { EnvironmentController } from './environmentController';
+import { getRepositoryInfoFromLocalFolder } from '../utils/utils';
 
 const ENVIRONMENT_CONFIG_NAME = 'environment';
 
 export class DocsEnvironmentController implements EnvironmentController, vscode.Disposable {
+
+    private activeWorkSpaceFolder: vscode.WorkspaceFolder;
     private environment: Environment;
     private configurationChangeListener: vscode.Disposable;
+    private _docsRepoType: DocsRepoType;

```

Comment:
should we keep the private property name style consistent?

Reply:
I'd very much like to, but that way `docsRepoType` would conflicts with `get docRepoType()`

Reply:
Created #78 to track